manifests/params.pp

```diff
@@ -283,6 +283,46 @@
           $maxpoll         = undef
           $disable_monitor = false
         }
+        'Amazon': {
```

Comment:
How weird. My local copy didn't have the duplication and showed no diffs/pull. Sorry. Updating now.